tests/unit/store/test_pickle_store.py

```diff
@@ -63,7 +69,8 @@ def test_read_object_2():
     assert PickleStore.read(self, arctic_lib, version, sentinel.symbol) == object
     assert coll.find.call_args_list == [call({'symbol': sentinel.symbol, 'parent': sentinel._id}, sort=[('segment', 1)])]
 
-
+@pytest.mark.xfail(sys.version_info >= (3,),
+                   reason="lz4 data written with python2 not compatible with python3")
```

Comment:
no the issue is bytes vs string in Python3. The file you have was encoded by LZ4 in python2 that returned a string encoding.